.mergify.yml

```diff
@@ -1,13 +1,12 @@
 pull_request_rules:
-  - name: Automatic merge
+  - name: Automatic merge to master
     conditions:
-      - "-draft"
-      - "#approved-reviews-by>=1"
-      - "approved-reviews-by=@gestalt-core"
```

Comment:
@chrislloyd what was the reason for removing this line? Don't we want to ensure PRs only get merged when `gestalt-core` approves?

Reply:
I didn't realize this before, but Mergify will respect branch protection settings that we've setup in GH: https://doc.mergify.io/actions.html#branch-protection-settings. The rules before were roughly approximating those. I could probably go further and remove the `approved-reviews-by` condition too.

Reply:
Actually I added these back in. Without it it'll _attempt to_ merge even if tests haven't passed which is a bit of a waste.